sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -1546,7 +1767,48 @@ private Mono<CosmosItemResponse<Object>> deleteAllItemsByPartitionKeyInternal(
                 requestOptions.getConsistencyLevel(),
                 OperationType.Delete,
                 ResourceType.PartitionKey,
-                client.getEffectiveDiagnosticsThresholds(requestOptions.getDiagnosticsThresholds()));
+                client.getEffectiveDiagnosticsThresholds(requestOptions.getDiagnosticsThresholds()),
+                null);
+    }
+
+    private <T> Mono<CosmosItemResponse<T>> replaceItemInternalCore(
+        Class<T> itemType,
+        String itemId,
+        Document doc,
+        CosmosItemRequestOptions options,
+        String trackingId) {
```

Comment:
feel trackingId does not need to be exposed in `replaceItemInternalCore`, it should already part of request options in up caller

Reply:
It is not part of COsmosxxxRequestOptions ( but a value generated by the caller / or null)